src/pages/SetPasswordPage.js

```diff
@@ -52,10 +53,9 @@ const defaultProps = {
     },
 };
 
-class SetPasswordPage extends Component {
+class SetPasswordPage extends React.Component {
```

Comment:
This was fine the way it was.

Reply:
Ahh my bad, all these seemingly unnecessary changes were from running the eslint check with `--fix`. Will get rid of them!

Reply:
I should just be able to revert that commit

Reply:
Cool cool thanks! Auto-fixers can sure make a mess.